css/style.css

```diff
@@ -578,13 +599,32 @@ section {
 }
 
 @media (max-width: 800px) {
+  aside {
+    width: 60%;
+  }
+
+  #sidebar ul li.active {
+    background-color: #ffffff !important;
+  }
+
   #navOptions li.active {
-    background-color: #ffffff;
-    padding-top: 15;
-    padding-bottom: 15;
+    padding-top: 14px;
+    padding-bottom: 14px;
+  }
+
+  #navOptions #who-li.active a {
```

Comment:
I believe you can write the next ten line with less duplication no?

Reply:
Because each section is a different color in the web view, I need to override each one manually on mobile. Otherwise the more specific color (from web) will override it. The "important" keyword also won't work because that overrides both active and inactive <a> tags. Besides that couldn't think of a way around it but open to suggestions!

Reply:
That's ok then